File: cloud/dynamic-site/gt-cs-1331-py/app/gtcs1331/solutions/solution112.py
'''Solution 1.12 implementation'''
from gtcs1331.solution_api.solution import Solution

class Solution112(Solution):
    '''Solution 1.12'''
    id = "1.12"
    title = "Speed Calculator"

    def __answer(self):
        speed = 24 / 1.6 / (1 + 40.0 / 60 + 35.0 / 60 / 60)

        return [f"{speed:.6f} kilometers per hour"]

    answer = property(__answer)

File: cloud/dynamic-site/gt-cs-1331-csharp/solutions/Solution112.cs
using _4MeOnWeb.SolutionAPI;

namespace _4MeOnWeb.Solutions {
    public class Solution112 : ImmutableSolution {
        public Solution112() :
            base("1.12", "Speed Calculator") {
        }

        public override IEnumerable<string> Answers {
            get {
                double speed = 24 / 1.6 / (1 + 40.0 / 60 + 35.0 / 60 / 60);

                return new[] { $"{speed:f6} kilometers per hour" };
            }
        }
    }
}